pkg/cmd/scan.go

```diff
@@ -168,6 +168,12 @@ func NewScanCmd(opts *pkg.ScanOptions) *cobra.Command {
 		"Terraform Cloud / Enterprise API token.\n"+
 			"Only used with tfstate+tfcloud backend.\n",
 	)
+	fl.StringVar(&opts.BackendOptions.TFCloudAPI,
+		"tfc-api",
```

Comment:
We should probably rename it to `tfc-endpoint`, I'm not sure that `tfc-api` is full of meanings for everyone.

Reply:
Could you also rename options naming in structs 🙏🏻 `s/TFCloudAPI/TFCloudEndpoint/` etc ...

Reply:
Yes.  Done.